lib/msal-node/src/crypto/HashUtils.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Hash } from "../utils/Constants";
+import crypto from "crypto";
+
+export class HashUtils {
+    /**
+     * generate 'SHA256' hash
+     * @param buffer
+     */
+    static sha256(buffer: string): Buffer {
```

Comment:
Can we make this a non-static method? Prefer to see us stop using static classes and methods for utility classes.

Reply:
Done, made it into an instantiable class and an instance method.